content/docs/user-guide/assets/asset-processor/settings.md

```diff
@@ -0,0 +1,46 @@
+---
+linkTitle: Settings
+title: Asset Processor Settings
+description: Commandline settings for AssetProcessor and AssetBuilder
+weight: 900
+toc: true
+---
+
+The Asset Processor and Asset Processor Batch expose a number of commandline settings that can be used when manually launching.
+
+For example, if you wanted to launch AssetProcessor manual on Windows but delay launch to attach a debugger:
+```
+AssetProcessor --project-path="C:/o3de/o3de/AutomatedTesting" --engine-path="C:/o3de/o3de" --waitOnLaunch
+```
+
+## General settings
+When manually launching Asset Processor or Asset Processor Batch you can set the following general properties:
+
+| Setting            | Description                                                         | Default                                               |
+|--------------------|---------------------------------------------------------------------|-------------------------------------------------------|
+| project-name       | Name of the current project.                                        | On Windows, to the current project's name             |
+| project-cache-path | Full path to the project cache folder.                              | On Windows, to the current project's cache path       |
+| project-path       | Supplies the path to the project that the AP should use (required). | On Windows, to the current project's path             |                                  
+| engine-path        | Supplies the engine path to the engine.                             | On Windows, to the engine registered with the project |
+
+
+## Advanced settings
+You can also control the behavior using the following command line options:
+
+| Setting                         | Description                                                                                                                                                                                                 | Notes                                                                                           |
```

Comment:
I think the "Notes" can be renamed as "Default" to match the above table. Also it makes the purpose clearer to me.